packages/core/src/components/Calendar/calendarClasses.tsx

```diff
@@ -0,0 +1,26 @@
+import { getClasses } from "utils";
+
+export type HvCalendarClasses = {
+  root?: string;
+  rangeCalendarContainer?: string;
+  singleCalendar?: string;
+  focusSelection?: string;
+  calendarMonthlyCell?: string;
+  calendarMonthlyCellSelected?: string;
+};
+
+const classKeys: string[] = [
+  "root",
+  "rangeCalendarContainer",
+  "singleCalendar",
+  "focusSelection",
+  "calendarMonthlyCell",
+  "calendarMonthlyCellSelected",
+];
+
+const calendarClasses = getClasses<HvCalendarClasses>(
+  classKeys,
+  "HvCalendarClasses"
```

Comment:
Just to maintain some consistency, I think we should use `HvCalendar` since we usually use the component's name here